baselines/dom.generated.d.ts

```diff
@@ -10439,18 +10439,20 @@ declare var StyleSheetPageList: {
 }
 
 interface SubtleCrypto {
-    decrypt(algorithm: string | Algorithm, key: CryptoKey, data: ArrayBufferView): any;
-    deriveBits(algorithm: string | Algorithm, baseKey: CryptoKey, length: number): any;
-    deriveKey(algorithm: string | Algorithm, baseKey: CryptoKey, derivedKeyType: string | Algorithm, extractable: boolean, keyUsages: string[]): any;
-    digest(algorithm: string | Algorithm, data: ArrayBufferView): any;
-    encrypt(algorithm: string | Algorithm, key: CryptoKey, data: ArrayBufferView): any;
-    exportKey(format: string, key: CryptoKey): any;
-    generateKey(algorithm: string | Algorithm, extractable: boolean, keyUsages: string[]): any;
-    importKey(format: string, keyData: ArrayBufferView, algorithm: string | Algorithm, extractable: boolean, keyUsages: string[]): any;
-    sign(algorithm: string | Algorithm, key: CryptoKey, data: ArrayBufferView): any;
-    unwrapKey(format: string, wrappedKey: ArrayBufferView, unwrappingKey: CryptoKey, unwrapAlgorithm: string | Algorithm, unwrappedKeyAlgorithm: string | Algorithm, extractable: boolean, keyUsages: string[]): any;
-    verify(algorithm: string | Algorithm, key: CryptoKey, signature: ArrayBufferView, data: ArrayBufferView): any;
-    wrapKey(format: string, key: CryptoKey, wrappingKey: CryptoKey, wrapAlgorithm: string | Algorithm): any;
+    decrypt(algorithm: string | RsaOaepParams | AesCtrParams | AesCbcParams | AesCmacParams | AesGcmParams | AesCfbParams, key: CryptoKey, data: ArrayBuffer | ArrayBufferView): PromiseLike<ArrayBuffer>;
```

Comment:
> not sure i see the point of enumerating all possible Algorithm types here

I'm not sure I fully understood the point of this comment until looking at it again now that we have the ability to write type aliases. Assuming you are asking "why are you listing all those xxxParams types instead of just using `AlgorithmIdentifier`?", the reason is that different algorithms have different allowed purposes, and I thought having them all enumerated would help users see at a glance which algorithms are usable for (e.g.) encrypting/decrypting and which are usable for signing/verifying.